content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -286,7 +286,8 @@ To cut a release:
 
 [source,shell]
 ----
-git checkout master
+# Checkout the primary branch of the repository
+git checkout $(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
```

Comment:
```suggestion
# Checkout the primary branch of the repository if you are on a UN*X system:
git checkout $(git symbolic-ref refs/remotes/origin/HEAD | sed 's@^refs/remotes/origin/@@')
# Windows users are adviced to use the following command in CMD:
# for /f "tokens=4 delims=/" %i in ('git symbolic-ref refs/remotes/origin/HEAD') do git checkout %i
```
I don't know powershell, but this works in plain cmd. Given with windows you have a plenty of shells, e.g. cmd, ps, git bash, wsl, etc. it's unreasonable to add examples for every occasion, when cmd would carry you through the process well.

Reply:
That meets my need and far beyond my need.  Thanks!